src/components/pagination/CdrPagination.vue

```diff
@@ -1,3 +1,170 @@
+
+<script setup>
+import { useCssModule, defineEmits, computed, ref, watch, nextTick, onMounted } from 'vue';
```

Comment:
Small nitpick, you don't need to import `defineEmits`, it's a compiler macro like `defineProps`